### PR TITLE
Editorial work throughout the ms

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

File: data_sharing_ms.md [hunk 1]
```diff
@@ -146,17 +149,17 @@ We provide three simple recommendations to help ensure that tabular data is prop
...
+While cross-tab data can be useful for its readability, and may be appropriate for data collection, this format makes it difficult to link the records with additional data (e.g., the location and environmental conditions at a site) and it cannot be properly used by most common database management and analysis tools (e.g., relational databases, dataframes in R and Python, etc.).
+If tabular data are currently in a cross-tab structure, there are tools to help restructure the data including functions in Excel, R (e.g., melt() function in the R package reshape [@wickham2007]), and Python (e.g., melt() function in the Pandas Python module [http://pandas.pydata.org/](http://pandas.pydata.org/)).
 
 In addition to following these basic rules you should also make sure to use descriptive column names [@borer2009].
-Descriptive column names can help to indicate what data is contained in each column and therefore make data interpretation errors less likely.
+Descriptive column names indicate make the data easier to understand and therefore make data interpretation errors less likely.
```
Comment (embaldridge) [inline data_sharing_ms.md]: "indicate" needs to be removed, but I am not entirely confident on my ability to use --amend yet.

**Comment by ethanwhite** [comment]
Thanks @embaldridge. I just fixed it.

**Comment by ethanwhite** [comment]
Any other thoughts on this?

**Comment by dmcglinn** [comment]
Ethan I like your edits. After these changes are merged I can try to tighten up the share your data section.

**Comment by ethanwhite** [comment]
Thanks Dan. I'm going to go ahead and merge this then and let you take a crack at the sharing data section.

**Pull request merged**